content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+do so, navigate to :menuselection:`... --> ...`.
+
+First, select the :guilabel:`UCFE Web Services` environment:
+
+- :guilabel:`Production`: for production databases. In this mode, electronic documents are sent to
+  |DGI| through Uruware for their validation.
+- :guilabel:`Testing`: for test databases. In this mode, the direct connection flows can be tested,
+  with the files sent to the |DGI| testing environment through Uruware.
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Credentials`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
+   :align: center
+
+.. note::
+   These credentials can be obtained from the email received when the :ref:`Uruguay account was
+   created <l10n_uy/uruware-account>`.
+
+Configure master data
+---------------------
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts <../accounting/get_started/chart_of_accounts>` is installed by default
+as part of the set of data included in the localization module, the accounts are mapped
+automatically in taxes, default accounts payable, and default accounts receivable.
+
+Accounts can by added or deleted according to the company's needs.
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+Contacts
+~~~~~~~~
+
+To create a contact, navigate to :menuselection:`Contacts app` and select :guilabel:`New`. Then
+enter the following information:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`: required to confirm an electronic invoice.
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`: required to confirm an electronic invoice.
+
+- :guilabel:`Identification Number`:
+
+  - :guilabel:`Type`: select a identification number type.
+  - :guilabel:`Number`: required to confirm an electronic invoice.
+
+Review taxes
+~~~~~~~~~~~~
+
+As part of the Uruguay localization module, taxes are automatically created with its configuration
+and related financial accounts.
+
+.. image:: uruguay/taxes.png
+   :align: center
+   :alt: Taxes for Uruguay.
+
+Review document types
+~~~~~~~~~~~~~~~~~~~~~
+
+Some accounting transactions like *Customer Invoices* and *Vendor Bills* are classified by document
+types. These are defined by the government fiscal authorities, in this case by the |DGI|.
+
+Each document type can have a unique sequence per journal where it is assigned. The data is created
+automatically when the localization module is installed, and the information required for the
+document types is included by default.
```

Comment:
Although this is true, we should add a note to specify that in Uruguay CAE are needed and have to be uploaded in Uruware. In Odoo we will get the sequence from Uruware based on their CAEs (works like in Chile, but in Uruguay they need to upload them in Uruware). CAEs are only used in production, in test they just need to set the range of sequences to be used in Uruware.

Reply:
We should add the following note @samueljlieber 

.. note::
   It is necessary to upload the document type sequence (known as CAE) in the Uruware portal. Odoo will get the sequence and PDF file from Uruware based on the CAEs. CAEs are only used in production. For the testing environment, a set range of sequences is needed. This is set in the testing portal from Uruware.